packages/bundle/src/core/location.ts

```diff
@@ -118,9 +118,15 @@ export const redirectToPage = async (redirect, meta) => {
     suggestion: meta.get('q'),
   });
 
+  if (redirect.get('url').startsWith('https')) {
+    document.location.href = redirect.get('url');
```

Comment:
@wolff95 we need to terminate function at this stage

Reply:
@wolff95 it will always start with `https`, and yes in that case no need to continue the function anymore.